### PR TITLE
adding profile pictures to frontend + database

### PR DESCRIPTION
**Pull request by lalvessiqueira**

### CODE REVIEW

File: backend/app.py [hunk 1]
```diff
@@ -101,19 +102,20 @@ def check_user():
         _json = request.get_json()
         _email = _json['email']
         _password = _json['password']
-        if (_email and _password):
-            user = db_operations.find_one({
-                'email': _email,
-                'password': _password
-            })
-            if user:
+        get_info = db_operations.find_one({
```
Comment (jpnoe) [inline backend/app.py:105]: should we still check if email is null?

File: backend/app.py [hunk 2]
```diff
@@ -123,6 +125,66 @@ def get_cache():
...
+        resp.status_code = 201
+        return resp
+    return not_found()
+
+#send file
+@app.route('/file/<filename>')
+def file(filename):
+    return mongo.send_file(filename)
+
+@app.route('/profile/<id>', methods=['GET'])
```
Comment (jpnoe) [inline backend/app.py:164]: rename url to avoid confusion

File: frontend/src/profile-form-FE/ImgUpload.jsx [hunk 3]
```diff
@@ -0,0 +1,92 @@
+import React, { Component } from "react";
```
Comment (jpnoe) [inline frontend/src/profile-form-FE/ImgUpload.jsx:1]: Looks good to me

File: frontend/src/profile-form-FE/ProfileImg.jsx [hunk 4]
```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+function ProfileImg(av) {
```
Comment (ericta-portfolio) [inline frontend/src/profile-form-FE/ProfileImg.jsx:3]: renaming av to props follow React naming conventions

**Review by ericta-portfolio — COMMENTED** [review]
Removing some comments and changing parameter names would add to cleanliness and style

File: backend/app.py [hunk 5]
```diff
@@ -1,9 +1,10 @@
 from flask import Flask
 from flask_pymongo import PyMongo
-# from bson.json_util import dumps
+from bson.json_util import dumps
 from bson.objectid import ObjectId
-from flask import jsonify, request
+from flask import jsonify, request, url_for
 from flask_cors import CORS
+from werkzeug.security import generate_password_hash, check_password_hash
```
Comment (pvolnuhi) [inline backend/app.py:8]: if there are some imports that aren't being used, just take them out to make the code cleaner.

same goes for commented out lines of code, instead could include a descriptive comment or just remove entirely.

File: backend/app.py [hunk 6]
```diff
@@ -123,6 +125,66 @@ def get_cache():
     CACHE["_id"] = str(CACHE["_id"])
     return jsonify(CACHE), 200
 
+@app.route('/upload/<id>', methods=['POST', 'PUT'])
+def upload(id):
+    if 'image' in request.files:
+        print("inside upload")
+        #create a file object
+        image = request.files['image']
+        _id = id
```
[the remark this answers is not in the document]
Comment (pvolnuhi) [reply backend/app.py:134]: I'd say make  an argument_id, as to not get confused with 'reusing' just id

File: frontend/src/profile-form-FE/ImgUpload.jsx (hunk 3 above)
Comment (pvolnuhi) [reply frontend/src/profile-form-FE/ImgUpload.jsx:1]: front-end looks clean, can remove unused comments and recheck for following React naming conventions as said below too.

**Pull request merged**